src/components/LoanCards/LoanProgressGroup.vue

```diff
@@ -3,7 +3,13 @@
 		<h4 class="tw-mb-0.5">
 			{{ fundingText }}
 		</h4>
+		<fundraising-status-meter
```

Comment:
What is the difference between `fundraising-status-meter` and `kv-progress-bar`? The loan cards look the same between this branch and dev on `lending-home`. If the components are basically the same, could the meter just replace the bar here? That would be ideal, since we could remove the new prop on the shared loan card component.

Reply:
They are slightly different, `FundraisingStatusMeter` has a darker grey background color while `KvProgressBar` has a lighter one (it matches the background color of the page). The Loan Card we are using is technically not supposed to be used anywhere else besides the homepage which has a background color that matches so we are doing some wonky stuff 😅 But when we revisit Loan Cards (after MVP) I think there is some room for tech debt/deletion/clean up

Reply:
(We'll also want to add this prop to the other sections of the MVP page so that it uses the correct progress bar - rfy + matched lending currently have the wrong one)

Reply:
I added this to the other components as well so this should be updated across the MVP page!